roles/foreman_installer/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: 'Run installer'
+  shell: >
+      {{ foreman_installer_command }}
+      --scenario {{ foreman_installer_scenario }}
+      {{ (foreman_installer_verbose|bool) | ternary("-v", "") }}
+      {{ (foreman_installer_no_colors|bool) | ternary("--no-colors", "") }}
+      {{ (foreman_installer_disable_system_checks|bool) | ternary("--disable-system-checks", "") }}
+      {{ foreman_installer_options | join(' ') }}
+  when: not foreman_installer_skip_installer
```

Comment:
With the current task list, I see no reason to expose `skip_installer` to the user?

Reply:
I had meant to ask you about that. Since this could be done at the role declaration level, whether we should translate this to here from Forklift or keep it clean (sounds like keep it clean).

Reply:
in forklift, we use that to speed up things as the role there is also taking care of installing packages, and we use "skip" to run install in parallel on server and proxy, and then without skip on server and then proxy, right?

sounds like a forklift oddity that should stay there.

Reply:
and even then, this could be optimized by having the install step in a install.yml and then using `import_role` with `tasks_from: install.yml`